test/test_utils_internal.py

```diff
@@ -13,9 +14,46 @@
 load_tests = load_tests
 
 from torch.testing._internal.common_utils import run_tests, TestCase
+from torch.testing._internal import fake_config_module
 
 
 class TestJustKnob(TestCase):
+    def test_justknob_config_module(self):
+        with self.subTest("saving+loading works correctly"):
+            fake_config_module.load_config(fake_config_module.save_config())
+        with self.subTest("portable converts to int"):
+            out = fake_config_module.save_config_portable()
+            self.assertEqual(out, {"example_knob_force": True})
+        with self.subTest("codegen config"):
+            out = fake_config_module.codegen_config()
+            self.assertEqual(out, "torch.testing._internal.fake_config_module.example_knob_force = True")
```

Comment:
There's no dependence between these tests. There's no reason to use subTest. Just make separate test methods for each of them if you really want to tell the difference

Reply:
Tests gone, but in general removed.